app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee

```diff
@@ -86,7 +86,11 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
   editDocument: (e) ->
     # Fire a global event. Consider refactoring this in the style of notifyExpandListeners above.
     Backbone.trigger 'LoanQuestionnairesView:edit', @
-
+    $('.view-element.answer')
+      .next('textarea')
+      .on 'change keyup keypress keydown paste cut', ->
+        $(this).height(0).height @scrollHeight
+      .change()
```

Comment:
I realize that this `change` trigger was in the code I found, but do you know if it's necessary?